thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/configs/AbstractConfig.java

```diff
@@ -21,6 +23,8 @@ public AbstractConfig() {
 
   public abstract String toJSON() throws Exception;
 
+  public abstract int getConfigId();
+
   public static <T extends AbstractConfig> T fromJSON(String json,
```

Comment:
This isn't something that would be constant just depending on the config_type. It comes from the object data